dev-libs/libprelude/libprelude-4.0.0.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_{4,5,6}} )
+
```

Comment:
Keep the eclass options together, no need for that much empty space.

Reply:
ok